.claude/skills/commit-msg/SKILL.md

```
@@ -0,0 +1,80 @@
+metadata:
+  author: shaharb
+  version: "1.0.0"
+---
+
+# Commit Message Generator
+
+Generate commit messages matching the pybandits project conventions.
+
+## Format
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)